frontend/src/components/common/NextButton.tsx

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 import { Button } from "@material-ui/core";
+import { red, green, blue, grey } from "@material-ui/core/colors";
 
 export const NextButton: React.FC = () => {
   return (
     <Button
       variant="contained"
       color="secondary"
-      style={{ marginTop: 12, marginBottom: 12 }}
+      style={{
```

Comment:
Using inline styles to this extent is generally considered bad practice, it would be better to convert this Button into a styled-component.